build.gradle.kts

```diff
@@ -19,44 +22,49 @@ plugins {
 
 allprojects {
   apply(plugin = "spinnaker.base-project")
+  if (property("enablePublishing") == "true") {
+    apply(plugin = "spinnaker.project")
+  }
 
   group = "com.netflix.spinnaker.keel"
 }
 
 subprojects {
-  apply(plugin = "nebula.kotlin")
-
-  repositories {
-    jcenter()
-  }
+  if (name != "keel-bom") {
```

Comment:
I hate this, but this stuff has to not be applied to the `keel-bom` project as it's not compatible with the `java-platform` plugin.